src/python/excel_uploader_main.py

```diff
@@ -257,13 +295,20 @@ def upload_driver(self, excel_file_path, gui_object):
         upload.upload(references, "Reference", self, gui_object)
 
         # Reassigns saved file nodes into their corresponding unsaved data nodes
-        for key, file in files.items():
-            data[key].files[0] = file
+        for key, vals in files.items():
+            for file in vals:
+                data[key].files.append(file)
 
         upload.upload(data, "Data", self, gui_object)
 
         upload.upload(materials, "Material", self, gui_object)
 
+        upload.upload(computations, "Computation", self, gui_object)
+
+        upload.upload(
```

Comment:
I think eventually if we put `upload.py` functions into `gui_main.py` that could make it a bit easier and cleaner